tests/console/rsync.pm

```diff
@@ -40,7 +40,8 @@ sub run {
     my $md5_initial_tar = script_output('md5sum /tmp/rsync_test_folder_a/rsync_test_tar.tar');
 
     enter_cmd("rsync -avzr /tmp/rsync_test_folder_a/ root\@localhost:/tmp/rsync_test_folder_b; echo \$\? > /tmp/rsync_return_code.txt");
```

Comment:
before this line, use `assert_script_run("ssh-keyscan localhost 127.0.0.1 ::1 | tee -a ~/.ssh/known_hosts");`

Reply:
this is not working yet, still checking for reason :)
no such file or directory... 

see https://openqa.suse.de/tests/9684651#step/rsync/33

Reply:
https://openqa.suse.de/tests/9684673#step/rsync/35 .ssh directory is not available, it means that ssh is not installed.